### ISSUE TITLE
django-ckeditor breaks SCRIPT_NAME parameter

### ISSUE THREAD

**Issue by pjrobertson** [django-ckeditor/django-ckeditor#726]
I've just spent a whole day debugging this. In short, when using ckeditor, it removes the SCRIPT_NAME (`get_script_prefix()` from all static/media URLs.

To reproduce:

This works:

1. Create a new project `django-admin startproject mysite` and `cd mysite`
2. Add a custom auth model mysite/models.py:
```
from django.db import models
from django.contrib.auth.models import AbstractBaseUser

class User(AbstractBaseUser):
    first_name = models.CharField('first name', max_length=30, blank=False)
    last_name = models.CharField('last name', max_length=30, blank=False)
    ui_language = models.CharField(max_length=5, null=True, verbose_name='interface language')
    USERNAME_FIELD = 'first_name'
```
3. In settings.py, add `mysite` to installed apps, and set `AUTH_USER_MODEL = 'mysite.User'`
4. Install `pip install gunicorn` (imagine we're getting ready to run this project in production)
5. Run gunicorn with `SCRIPT_NAME=/backend gunicorn sa_dash.wsgi:application --bind 127.0.0.1:8000`
6. Go to `http://127.0.0.1:8000/backend/admin/login/` - verify that the page loads (the static files won't load, that's fine)
7. View the page source and verify that the static URLs are also prepended with the SCRIPT_NAME (`/backend`) as we expect

This doesn't work:

8. Now, install ckeditor `pip install django-ckeditor`
9. In models.py, simply add the line: `from ckeditor.fields import RichTextField`
10. Re-run gunicorn with `SCRIPT_NAME=/backend gunicorn sa_dash.wsgi:application --bind 127.0.0.1:8000`
11. Load up `http://127.0.0.1:8000/backend/admin/login/` - note how the static 

-------

The cause: in widgets.py, you shouldn't be importing `from django.templatetags.static import static` like this. It means that the static info is all potentially loaded before the app has actually loaded.

I'm not sure if that can just be removed or not

**Comment by pjrobertson** [comment]
OK, so the easy way to fix it is to set `CKEDITOR_BASEPATH` in settings.py - that way, the `static()` method never gets called. But that's a bit of a hack

**Comment by matthiask** [comment]
Can you try installing the current `master` branch version and check if this still happens? Thanks!

**Comment by pjrobertson** [comment]
Confirmed fixed with `ckeditor@master`.

Thanks for the speedy response @matthiask !!

Looking forward to seeing this released :)

**Comment by matthiask** [comment]
Thanks for testing! Released as `django-ckeditor==6.5.0`

Fixed in e2d3dfa

**Comment by pjrobertson** [comment]
🚀🚀🚀🚀🚀
